File: pkg/runtime/runtimes/java/com/bitmechanic/pulserpc/VerificationResult.java
package com.bitmechanic.pulserpc;

import java.util.List;

public class VerificationResult {
    private final boolean compatible;
    private final String serverChecksum;
    private final String clientChecksum;
    private final List<ContractDelta> deltas;
    private final long timestamp;

    public VerificationResult(boolean compatible, String serverChecksum, String clientChecksum,
                              List<ContractDelta> deltas, long timestamp) {
        this.compatible = compatible;
        this.serverChecksum = serverChecksum;
        this.clientChecksum = clientChecksum;
        this.deltas = deltas;
        this.timestamp = timestamp;
    }

    public boolean isCompatible() { return compatible; }
    public String getServerChecksum() { return serverChecksum; }
    public String getClientChecksum() { return clientChecksum; }
    public List<ContractDelta> getDeltas() { return deltas; }
    public long getTimestamp() { return timestamp; }
}

File: pkg/runtime/runtimes/csharp/PulseRPC/VerificationResult.cs
using System;
using System.Collections.Generic;

namespace PulseRPC
{
    public class VerificationResult
    {
        public bool Compatible { get; init; }
        public string ServerChecksum { get; init; } = string.Empty;
        public string ClientChecksum { get; init; } = string.Empty;
        public IReadOnlyList<ContractDelta> Deltas { get; init; } = Array.Empty<ContractDelta>();
        public DateTime Timestamp { get; init; }
    }
}